engine.py

```diff
@@ -55,6 +55,28 @@ def tank_mobu_exception_trap(ex_cls, ex, tb):
 
 
 class MotionBuilderEngine(tank.platform.Engine):
+
+    @property
+    def host_info(self):
+        """
+        :returns: A {"name": application name, "version": application version}
```

Comment:
Same comment here as mentioned in the `tk-mari` PR about the docstring formatting. 

https://github.com/shotgunsoftware/tk-mari/pull/6#pullrequestreview-63355284